pkg/config/jobs.go

```diff
@@ -266,6 +266,17 @@ type Postsubmit struct {
 	JenkinsSpec *JenkinsSpec `json:"jenkins_spec,omitempty"`
 }
 
+// RetriggerFailedRun defines the configuration for retrying failed prowjobs.
+type RetriggerFailedRun struct {
+	// UntilSuccess stops retries once a successful run is achieved.
+	// The Attempts field still limits the total number of retries.
+	UntilSuccess bool
+	// Attempts specifies the maximum number of retry attempts allowed.
+	Attempts int
```

Comment:
Number of attempts also sounds like something that should be bounded somehow. Maybe a horologium config that would limit this, no matter what the job configured?